common/device-model.cpp

```diff
@@ -3349,21 +3349,40 @@ namespace rs2
         {
             if (sub->supports_on_chip_calib() && !has_autocalib)
             {
+                bool is_d555 = false;
+
+                if( dev.supports( RS2_CAMERA_INFO_PRODUCT_ID ) )
+                {
+                    auto pid_str = std::string( dev.get_info( RS2_CAMERA_INFO_PRODUCT_ID ) );
+                    if( pid_str == "0B56" || pid_str == "DDS" )
+                        is_d555 = true;
+                }
+
                 if (ImGui::Selectable("On-Chip Calibration", false, avoid_selection_flag))
                 {
                     try
                     {
-                        auto manager = std::make_shared<d500_on_chip_calib_manager>(viewer, sub, *this, dev);
-                        auto n = std::make_shared<d500_autocalib_notification_model>("", manager, false);
-                        viewer.not_model->add_notification(n);
+                        std::shared_ptr< process_manager > manager;
+                        std::shared_ptr< process_notification_model > n;
+                        if( is_d555 )
+                        {
+                            manager = std::make_shared< on_chip_calib_manager >( viewer, sub, *this, dev );
```

Comment:
This is a bit confusing (naming).
D555 is also d500
We have d555 or d500

Reply:
This whole function is about D500 products (draw_device_panel_auto_calib_**d500**). `is_d555` is a special case for D500. I am open to new name suggestions...

Reply:
The naming issue is this:


n D555 you use on_chip_calib_manager and on other you use d500_on_chip_calib_manager 
D555 is D500 too..

It's a bit confusing, but not a must to change


    if( is_d555 )
     {
         manager = std::make_shared< on_chip_calib_manager >( viewer, sub, *this, dev );
       }
      else
             manager = std::make_shared< d500_on_chip_calib_manager >( viewer, sub, *this,

Reply:
I can add a comment that since D555 shares OHM with D455 we run the D400 algorithms for it

Reply:
Sounds good..